resolve javadoc warnings

In the application code (`mvn javadoc:javadoc`)

```
22 warnings
[WARNING] Javadoc Warnings
[WARNING] javadoc: warning - Multiple sources of package comments found for package "org.eclipse.emf.common"
[WARNING] javadoc: warning - Multiple sources of package comments found for package "org.eclipse.emf.common.util"
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/util/IPAuthenticationFilter.java:162: warning: no description for @return
[WARNING] * @return
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/util/IPAuthenticationFilter.java:186: warning: no description for @param
[WARNING] * @param filterConfig
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @param for f
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @throws for java.io.IOException
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @throws for javax.xml.transform.TransformerConfigurationException
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @throws for javax.xml.transform.TransformerException
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @throws for javax.xml.parsers.ParserConfigurationException
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @throws for org.xml.sax.SAXException
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @throws for org.opengis.referencing.operation.TransformException
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:136: warning: no @throws for org.opengis.referencing.FactoryException
[WARNING] public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:350: warning: no @param for gd
[WARNING] public static AttributeDescriptor changeGeometryBinding(GeometryDescriptor gd, Class geomBinding) {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:350: warning: no @param for geomBinding
[WARNING] public static AttributeDescriptor changeGeometryBinding(GeometryDescriptor gd, Class geomBinding) {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:350: warning: no @return
[WARNING] public static AttributeDescriptor changeGeometryBinding(GeometryDescriptor gd, Class geomBinding) {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:84: warning: no @param for workingDir
[WARNING] public StreamingShapeWriter(String workingDir) throws IOException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:84: warning: no @throws for java.io.IOException
[WARNING] public StreamingShapeWriter(String workingDir) throws IOException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:111: warning: no @throws for java.io.IOException
[WARNING] public StreamingShapeWriter(String workingDir, boolean shapeIndex) throws IOException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:98: warning: no @throws for java.io.IOException
[WARNING] public StreamingShapeWriter(String workingDir, boolean shapeIndex, List<String> skipAttributeNames) throws IOException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:76: warning: no @param for workingDir
[WARNING] public StreamingShapeWriter(String workingDir, CoordinateReferenceSystem defaultCoordRefSys) throws IOException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:76: warning: no @param for defaultCoordRefSys
[WARNING] public StreamingShapeWriter(String workingDir, CoordinateReferenceSystem defaultCoordRefSys) throws IOException {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java:76: warning: no @throws for java.io.IOException
[WARNING] public StreamingShapeWriter(String workingDir, CoordinateReferenceSystem defaultCoordRefSys) throws IOException {
[WARNING] ^
```

and in the test code (`mvn javadoc:test-javadoc`)

```
3 warnings
[WARNING] Javadoc Warnings
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/test/java/nl/b3p/viewer/stripes/AttributesActionBeanTest.java:177: warning: no @throws for java.lang.Exception
[WARNING] public void testStoreFirstPageGS() throws Exception {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/test/java/nl/b3p/viewer/stripes/AttributesActionBeanTest.java:210: warning: no @throws for java.lang.Exception
[WARNING] public void testStoreFirstPageDG() throws Exception {
[WARNING] ^
[WARNING] /mnt/tera1/home/mark/dev/projects/flamingo/viewer/src/test/java/nl/b3p/viewer/util/FeatureToJsonTest.java:63: warning: no @throws for java.lang.Exception
[WARNING] public void testGetJSONFeaturesFirstPage() throws Exception {
[WARNING] ^
```

--- a/pom.xml
+++ b/pom.xml
@@ -196,7 +196,7 @@
                 <version>1.2.4</version>
                 <exclusions>
                     <exclusion>
-                        <!-- drags in servlet api 2.5-->
+                        <!-- drags in servlet api 2.5 -->
                         <groupId>javax.servlet</groupId>
                         <artifactId>servlet-api</artifactId>
                     </exclusion>
@@ -205,6 +205,12 @@
                         <artifactId>jsp-api</artifactId>
                     </exclusion>
                 </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>javax</groupId>
+                <artifactId>javaee-endorsed-api</artifactId>
+                <version>6.0</version>
+                <scope>provided</scope>
             </dependency>
             <!-- geozet openls -->
             <dependency>

--- a/viewer-admin/pom.xml
+++ b/viewer-admin/pom.xml
@@ -61,7 +61,10 @@
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>javax</groupId>
+            <artifactId>javaee-endorsed-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>javax.mail</groupId>
             <artifactId>javax.mail-api</artifactId>

--- a/viewer-commons/src/main/java/org/opensphere/geometry/algorithm/ConcaveHull.java
+++ b/viewer-commons/src/main/java/org/opensphere/geometry/algorithm/ConcaveHull.java
@@ -51,7 +51,7 @@ import com.vividsolutions.jts.triangulate.quadedge.QuadEdge;
 import com.vividsolutions.jts.triangulate.quadedge.QuadEdgeSubdivision;
 import com.vividsolutions.jts.triangulate.quadedge.QuadEdgeTriangle;
 import com.vividsolutions.jts.util.UniqueCoordinateArrayFilter;
-import org.openshere.geometry.triangulation.DoubleComparator;
+import org.opensphere.geometry.triangulation.DoubleComparator;
 
 
 /**
@@ -89,10 +89,10 @@ public class ConcaveHull {
 
 	/**
 	 * Create a new concave hull construction for the input {@link Geometry}.
-	 * 
-	 * @param geometry
-	 * @param threshold
-	 */
+	 *
+     * @param geometry the geometry
+     * @param threshold the maximum length of border edges of the concave hull
+     */
 	public ConcaveHull(Geometry geometry, double threshold) {
 		this.geometries = transformIntoPointGeometryCollection(geometry);
 		this.threshold = threshold;
@@ -101,10 +101,10 @@ public class ConcaveHull {
 	
 	/**
 	 * Create a new concave hull construction for the input {@link GeometryCollection}.
-	 * 
-	 * @param geometries
-	 * @param threshold
-	 */
+	 *
+     * @param geometries collection of geometries
+     * @param threshold the maximum length of border edges of the concave hull
+     */
 	public ConcaveHull(GeometryCollection geometries, double threshold) {
 		this.geometries = transformIntoPointGeometryCollection(geometries);
 		this.threshold = threshold;

--- a/viewer-commons/src/main/java/org/opensphere/geometry/triangulation/DoubleComparator.java
+++ b/viewer-commons/src/main/java/org/opensphere/geometry/triangulation/DoubleComparator.java
@@ -22,7 +22,7 @@
  * Eric Grosso, eric.grosso.os@gmail.com
  * 
  */
-package org.openshere.geometry.triangulation;
+package org.opensphere.geometry.triangulation;
 
 import java.util.Comparator;
 import java.util.Map;
@@ -57,12 +57,9 @@ public class DoubleComparator implements Comparator<QuadEdge> {
 	 * 		quad edge to compare
 	 * @param qeB
 	 * 		quad edge to compare
-	 * @return
-	 * 		1 if double value associated to qeA  < double
-	 * 		value associated to qeB,
-	 * 		0 if values are equals,
-	 * 		-1 otherwise
-	 */
+	 * @return 1 if double value associated to qeA &lt; double value associated
+     * to qeB, 0 if values are equals, -1 otherwise
+   	 */
 	@Override
 	public int compare(QuadEdge qeA, QuadEdge qeB) {
 		if (this.map.get(qeA) < this.map.get(qeB)) {

--- a/viewer-commons/src/main/java/org/opensphere/geometry/triangulation/model/Edge.java
+++ b/viewer-commons/src/main/java/org/opensphere/geometry/triangulation/model/Edge.java
@@ -292,7 +292,8 @@ public class Edge {
 	/**
 	 * Add a triangle in relationship with the edge.
 	 * 
-	 * @return
+     * @param triangle the triangle to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addTriangle(Triangle triangle) {
@@ -302,7 +303,8 @@ public class Edge {
 	/**
 	 * Add triangles in relationship with the edge.
 	 * 
-	 * @return
+     * @param triangles the triangles to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addTriangles(List<Triangle> triangles) {
@@ -312,7 +314,8 @@ public class Edge {
 	/**
 	 * Remove a triangle in relationship with the edge.
 	 * 
-	 * @return
+     * @param triangle the triangle to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeTriangle(Triangle triangle) {
@@ -322,7 +325,8 @@ public class Edge {
 	/**
 	 * Remove triangles in relationship with the edge.
 	 * 
-	 * @return
+     * @param triangles the triangles to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeTriangles(List<Triangle> triangles) {
@@ -332,7 +336,8 @@ public class Edge {
 	/**
 	 * Add an incident edge in relationship with the edge.
 	 * 
-	 * @return
+     * @param edge incident edge to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addIncidentEdge(Edge edge) {
@@ -342,7 +347,8 @@ public class Edge {
 	/**
 	 * Add incident edges in relationship with the edge.
 	 * 
-	 * @return
+     * @param edges incident edges to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addIncidentEdges(List<Edge> edges) {
@@ -352,7 +358,8 @@ public class Edge {
 	/**
 	 * Remove an incident edge in relationship with the edge.
 	 * 
-	 * @return
+     * @param edge incident edge to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeIncidentEdge(Edge edge) {
@@ -362,7 +369,8 @@ public class Edge {
 	/**
 	 * Remove incident edges in relationship with the edge.
 	 * 
-	 * @return
+     * @param edges incident edges to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeAllIncidentEdges(List<Edge> edges) {

--- a/viewer-commons/src/main/java/org/opensphere/geometry/triangulation/model/Triangle.java
+++ b/viewer-commons/src/main/java/org/opensphere/geometry/triangulation/model/Triangle.java
@@ -169,7 +169,8 @@ public class Triangle {
 	/**
 	 * Add an edge to the triangle.
 	 * 
-	 * @return
+     * @param edge the edge to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addEdge(Edge edge) {
@@ -178,8 +179,8 @@ public class Triangle {
 
 	/**
 	 * Add edges to the triangle.
-	 * 
-	 * @return
+	 * @param edges the edges to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addEdges(List<Edge> edges) {
@@ -188,8 +189,8 @@ public class Triangle {
 
 	/**
 	 * Remove an edge of the triangle.
-	 * 
-	 * @return
+	 * @param edge the edge to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeEdge(Edge edge) {
@@ -198,8 +199,9 @@ public class Triangle {
 
 	/**
 	 * Remove edges of the triangle.
-	 * 
-	 * @return
+	 *
+     * @param edges the edges to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeEdges(List<Edge> edges) {
@@ -210,7 +212,8 @@ public class Triangle {
 	/**
 	 * Add a neighbour triangle to the triangle.
 	 * 
-	 * @return
+     * @param triangle neighbour to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addNeighbour(Triangle triangle) {
@@ -220,7 +223,8 @@ public class Triangle {
 	/**
 	 * Add neighbour triangles to the triangle.
 	 * 
-	 * @return
+     * @param triangles neighbours to add
+   	 * @return
 	 * 		true if added, false otherwise
 	 */
 	public boolean addNeighbours(List<Triangle> triangles) {
@@ -230,7 +234,8 @@ public class Triangle {
 	/**
 	 * Remove a neighbour triangle of the triangle.
 	 * 
-	 * @return
+     * @param triangle neighbour to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeNeighbour(Triangle triangle) {
@@ -240,7 +245,8 @@ public class Triangle {
 	/**
 	 * Remove neighbour triangles of the triangle.
 	 * 
-	 * @return
+     * @param triangles neighbours to remove
+   	 * @return
 	 * 		true if removed, false otherwise
 	 */
 	public boolean removeNeighbours(List<Triangle> triangles) {

--- a/viewer/pom.xml
+++ b/viewer/pom.xml
@@ -128,6 +128,10 @@
             <groupId>org.glassfish.web</groupId>
             <artifactId>javax.servlet.jsp.jstl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax</groupId>
+            <artifactId>javaee-endorsed-api</artifactId>
+        </dependency>
         <!-- geozet openls -->
         <dependency>
             <groupId>nl.geozet</groupId>

--- a/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java
+++ b/viewer/src/main/java/nl/b3p/viewer/features/StreamingShapeWriter.java
@@ -71,7 +71,11 @@ public class StreamingShapeWriter {
     private CoordinateReferenceSystem defaultCoordRefSys = null;
 
     /**
-     * Constructor with shapeIndex=true
+     * Constructor with shapeIndex=true.
+     *
+     * @param workingDir The working dir where the shape writer is going to
+     * write the shape files
+     * @param defaultCoordRefSys output CRS
      */
     public StreamingShapeWriter(String workingDir, CoordinateReferenceSystem defaultCoordRefSys) throws IOException {
         this(workingDir);
@@ -79,7 +83,12 @@ public class StreamingShapeWriter {
     }
 
     /**
-     * Constructor with shapeIndex=true
+     * Constructor with shapeIndex=true.
+     *
+     * @param workingDir The working dir where the shape writer is going to
+     * write the shape files
+     * @throws java.io.IOException if the working dir is not writable or does
+     * not exist
      */
     public StreamingShapeWriter(String workingDir) throws IOException {
         this(workingDir, true);
@@ -94,6 +103,8 @@ public class StreamingShapeWriter {
      * @param shapeIndex If true a index wil be created on the shape files.
      * @param skipAttributeNames a list of attribute names that needs to be
      * skipped (default only boundedBy)
+     * @throws java.io.IOException if the working dir is not writable or does
+     * not exist
      */
     public StreamingShapeWriter(String workingDir, boolean shapeIndex, List<String> skipAttributeNames) throws IOException {
         this(workingDir, shapeIndex);
@@ -107,6 +118,8 @@ public class StreamingShapeWriter {
      * @param workingDir The working dir where the shape writer is going to
      * write the shape files
      * @param shapeIndex If true a index wil be created on the shape files.
+     * @throws java.io.IOException if the working dir is not writable or does
+     * not exist
      */
     public StreamingShapeWriter(String workingDir, boolean shapeIndex) throws IOException {
         props.put(ShapefileDataStoreFactory.URLP.key, "");
@@ -130,10 +143,20 @@ public class StreamingShapeWriter {
     }
 
     /**
-     * Writes a feature to a shape file. It also seperates the different
+     * Writes a feature to a shape file. It also separates the different
      * GeometryTypes into different shapefiles.
+     *
+     * @param f feature to write
+     * @throws java.io.IOException if writing fails
+     * @throws org.xml.sax.SAXException if parsing fails
+     * @throws javax.xml.transform.TransformerConfigurationException if parsing
+     * fails
+     * @throws javax.xml.parsers.ParserConfigurationException if parsing fails
+     * @throws org.opengis.referencing.operation.TransformException if
+     * transforming fails
+     * @throws org.opengis.referencing.FactoryException if factory lookup fails
      */
-    public void write(SimpleFeature f) throws IOException, TransformerConfigurationException, TransformerException, ParserConfigurationException, SAXException, TransformException, FactoryException {
+    public void write(SimpleFeature f) throws IOException, SAXException, TransformerConfigurationException, TransformerException, ParserConfigurationException, TransformException, FactoryException {
 
         featuresGiven++;
         Class newGeomClass = null;
@@ -274,8 +297,8 @@ public class StreamingShapeWriter {
      * @param suffix A suffix after the shapefile name to indicate the
      * difference between geometry types. For example "_p" (for points)
      * @return a new featurewriter
-     * @throws MalformedURLException
-     * @throws IOException
+     * @throws MalformedURLException if any
+     * @throws IOException if any
      */
     private FeatureWriter createNewWriter(Class geomClass, SimpleFeatureType type, String suffix) throws MalformedURLException, IOException {
         //create the file path
@@ -346,6 +369,9 @@ public class StreamingShapeWriter {
 
     /**
      * Function to change the geometryBinding of a GeometryDescriptor
+     * @param gd geometry descriptor
+     * @param geomBinding geometry class
+     * @return the new geometry descriptor
      */
     public static AttributeDescriptor changeGeometryBinding(GeometryDescriptor gd, Class geomBinding) {
         AttributeTypeBuilder builder = new AttributeTypeBuilder();

--- a/viewer/src/main/java/nl/b3p/viewer/stripes/ApplicationActionBean.java
+++ b/viewer/src/main/java/nl/b3p/viewer/stripes/ApplicationActionBean.java
@@ -33,7 +33,6 @@ import net.sourceforge.stripes.validation.SimpleError;
 import net.sourceforge.stripes.validation.Validate;
 import nl.b3p.viewer.components.ComponentRegistry;
 import nl.b3p.viewer.components.ComponentRegistryInitializer;
-import nl.b3p.viewer.components.ViewerComponent;
 import nl.b3p.viewer.config.ClobElement;
 import org.stripesstuff.stripersist.Stripersist;
 import nl.b3p.viewer.config.app.Application;
@@ -342,6 +341,7 @@ public class ApplicationActionBean implements ActionBean {
      * meant to be a secure hash, the roles of a user are not secret.
      *
      * @param request servlet request with user credential
+     * @param em the entitymanahger to use for database access
      * @return a key to use as a cache identifyer
      */
     public static int getRolesCachekey(HttpServletRequest request, EntityManager em) {

--- a/viewer/src/main/java/nl/b3p/viewer/util/FeatureToJson.java
+++ b/viewer/src/main/java/nl/b3p/viewer/util/FeatureToJson.java
@@ -103,6 +103,9 @@ public class FeatureToJson {
 
     /**
      *
+     * @param arrays {@code true} when???
+     * @param edit {@code true} to return editable data
+     * @param graph {@code true} when??
      * @param aliases {@code true} to return aliases in the response
      * @param returnNullval {@code true} to return empty string for null value
      * @param attributesToInclude List of attributes to return in the response

--- a/viewer/src/main/java/nl/b3p/viewer/util/IPAuthenticationFilter.java
+++ b/viewer/src/main/java/nl/b3p/viewer/util/IPAuthenticationFilter.java
@@ -73,6 +73,7 @@ public class IPAuthenticationFilter implements Filter {
      * @exception IOException if an input/output error occurs
      * @exception ServletException if a servlet error occurs
      */
+    @Override
     public void doFilter(ServletRequest r, ServletResponse response,
             FilterChain chain)
             throws IOException, ServletException {
@@ -159,7 +160,7 @@ public class IPAuthenticationFilter implements Filter {
 
     /**
      * Return the filter configuration object for this filter.
-     * @return 
+     * @return filter configuration object
      */
     public FilterConfig getFilterConfig() {
         return (this.filterConfig);
@@ -182,8 +183,9 @@ public class IPAuthenticationFilter implements Filter {
     }
 
     /**
-     * Init method for this filter
-     * @param filterConfig
+     * Init method for this filter.
+     *
+     * @param filterConfig filter configuration object
      */
     @Override
     public void init(FilterConfig filterConfig) {        

--- a/viewer/src/test/java/nl/b3p/viewer/stripes/AttributesActionBeanTest.java
+++ b/viewer/src/test/java/nl/b3p/viewer/stripes/AttributesActionBeanTest.java
@@ -173,6 +173,7 @@ public class AttributesActionBeanTest extends TestUtil{
 
     /**
      * Test of store method, of class AttributesActionBean.
+     * @throws java.lang.Exception if any
      */
     @Test
     public void testStoreFirstPageGS() throws Exception {
@@ -188,7 +189,12 @@ public class AttributesActionBeanTest extends TestUtil{
         JSONArray features = result.getJSONArray("features");
         assertEquals(10, features.length());
     }
-    
+
+    /**
+     * Test of store method, of class AttributesActionBean.
+     *
+     * @throws Exception if any
+     */
     @Test
     public void testStoreSecondPageGS() throws Exception {
         System.out.println("store");
@@ -206,6 +212,7 @@ public class AttributesActionBeanTest extends TestUtil{
     
     /**
      * Test of store method, of class AttributesActionBean.
+     * @throws java.lang.Exception if any
      */
     @Test //at the time of committing, the service was down.
     public void testStoreFirstPageDG() throws Exception {
@@ -222,6 +229,11 @@ public class AttributesActionBeanTest extends TestUtil{
         assertEquals(10, features.length());
     }
     
+    /**
+     * Test of store method, of class AttributesActionBean.
+     *
+     * @throws Exception if any
+     */
     @Test //at the time of committing, the service was down.
     public void testStoreSecondPageDG() throws Exception {
         System.out.println("store");
@@ -237,6 +249,10 @@ public class AttributesActionBeanTest extends TestUtil{
         assertEquals(10, features.length());
     }
     
+    /**
+     *
+     * @throws Exception if any
+     */
     @Test
     public void testGetJSONFeaturesRelatedFeaturetypeSub() throws Exception {
         System.out.println("store");
@@ -256,6 +272,10 @@ public class AttributesActionBeanTest extends TestUtil{
         assertEquals(10, features.length());
     }
     
+    /**
+     *
+     * @throws Exception if any
+     */
     @Test
     public void testGetJSONFeaturesRelatedFeaturetypeMain() throws Exception {
         initData(false);

--- a/viewer/src/test/java/nl/b3p/viewer/util/FeatureToJsonTest.java
+++ b/viewer/src/test/java/nl/b3p/viewer/util/FeatureToJsonTest.java
@@ -58,6 +58,7 @@ public class FeatureToJsonTest {
 
     /**
      * Test of getJSONFeatures method, of class FeatureToJson.
+     * @throws java.lang.Exception if any
      */
     @Test
     public void testGetJSONFeaturesFirstPage() throws Exception {

